packages/react-scripts/package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "prettier": "npx prettier --arrow-parens always --single-quote --trailing-comma es5 --print-width 120 --no-semi --write \"template*/**/*.js\"",
     "fs-publish": "npmPublish",
-    "prepare": "modernizr -c modernizr-config.json -d layout"
+    "prepublishOnly": "npx modernizr -c modernizr-config.json -d layout"
```

Comment:
so we only need modernizr to run when publishing?

Reply:
Yes, this was based on working through this with Joey.